mgmtsystem_claim/views/res_partner_views.xml

```diff
@@ -13,7 +13,7 @@
                     context="{'search_default_partner_id': active_id, 'default_partner_id': active_id}"
                     name="%(mgmtsystem_claim_action)d"
                     icon="fa-comments"
-                    groups="sales_team.group_sale_salesman,base.group_partner_manager"
+                    groups="sales_team.group_sale_salesman"
```

Comment:
This change must be moved in the migration commit

Reply:
Yes I know, but I'm lazy ;-)

Reply:
Still, he makes a good point and it should be fixed 😅 
It's currently in the `[DONT MERGE] test-requirements.txt` commit, which will be removed after the dependency is merged. We don't want to loose these changes by mistake so better to put them in the right commit 😄

Reply:
ok guys, done